src/raft.rs

```diff
@@ -380,6 +386,12 @@ impl<T: Storage> Raft<T> {
         self.skip_bcast_commit = skip;
     }
 
+    /// Returns the inner raft group configuration
+    #[inline]
+    pub fn groups(&self) -> &Groups {
```

Comment:
I think it's better to make `Groups` be private for external view. So I prefer `pub fn groups(&self) -> HashMap<group-id, Vec<peer-id>>`.